sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -181,8 +187,13 @@ def delete_key(self, name, **kwargs):
                 :caption: Delete a key
                 :dedent: 8
         """
-        bundle = self._client.delete_key(self.vault_endpoint, name, error_map=_error_map, **kwargs)
-        return DeletedKey._from_deleted_key_bundle(bundle)
+        polling_interval = kwargs.pop("_polling_interval", 2)
```

Comment:
Seems like `polling_interval` would be an acceptable keyword, what's the rationale for trying to (further) hide the argument?

Reply:
Previously talked to Laurent about this, he said to keep it _ so we don't have to keep supporting it